docs/features/feature-guides/compliance-forms/complete-a-form.md

```diff
@@ -0,0 +1,176 @@
+---
+title: Complete a Form
+---
+
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# Complete a DataHub Compliance Form
+<FeatureAvailability/>
+
+This guide provides an example of completing a Compliance Form, covering:
+
+1. Accessing a Form from an Asset Page or the DataHub Cloud Task Center
+2. Completing a Form for a single Asset or multiple Assets (DataHub Cloud only)
+3. Understanding different Form Question completion states
+
+The example uses the **Governance Initiative 2024**, a Verification Form with 3 Required Questions:
+
+<p align="center">
+  <img
+       width="90%"
+       src="https://raw.githubusercontent.com/datahub-project/static-assets/refs/heads/main/imgs/compliance_forms/sample-compliance-form.png" 
+       alt="Sample Compliance Form"/>
+</p>
+
+## Accessing a Compliance Form
```

Comment:
Now the tone changes to "active" where you are putting "ing" at the end of the action. For the all other places you choose not to do this .. I would recommend keeping it consistent so users can easily orient themselves in the docs

Reply:
good catch